tests/security/selinux/selinux_setup.pm

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup environment for selinux test
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#40358
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use version_utils "is_opensuse";
+
+sub run {
+    select_console "root-console";
+
+    if (is_opensuse) {
+        zypper_call("ar http://download.opensuse.org/tumbleweed/repo/oss/ tumbleweed-Oss");
+    }
+
+    # program 'sestatus' can be found in policycoreutils pkgs
+    zypper_call("--no-refresh in policycoreutils");
+
+    my $pkgs = script_output("echo `zypper --no-refresh se selinux | grep -i selinux | grep -v -w srcpackage | cut -d '|' -f 2`");
+    zypper_call("--no-refresh in  $pkgs", timeout => 6000);
+
+    # for opensuse install policy pkgs as needed
+    if (is_opensuse) {
+        my @files = ("selinux-policy-20140730-103.3.noarch.rpm");
```

Comment:
yeah maybe one day you will get list of files here , and **that** would be the day one you will add loop here. But now you have single file so behave accordingly :
```
my $file="selinux-policy-20140730-103.3.noarch.rpm";
assert_script_run "wget --quiet " . data_url("selinux/$file");
assert_script_run("rpm -ivh --nosignature $file");
```

Reply:
Done! thanks.
(Yeah, I planed to added more files)